packages/pigment-css-vite-plugin/src/index.ts

```diff
@@ -41,9 +41,11 @@ function isZeroRuntimeProcessableFile(fileName: string, transformLibraries: stri
   );
 }
 
-export function pigment(options: PigmentOptions) {
+const defaultTheme = extendTheme({});
+
+export function pigment(options?: PigmentOptions) {
   const {
-    theme,
+    theme = defaultTheme,
```

Comment:
```suggestion
    theme = {},
```

An empty object might be a better default since the behaviour changes slightly when theme is a plain object vs when it's created using `extendTheme`. I'd rather users get the plain experience by default unless they opt-in manually.

Reply:
I was having the same thought - this is why I added in my duplicated PR https://github.com/mui/material-ui/pull/41692, however I see the value e.g. for people having the `applyStyles` function and similar utils we may introduce in the future, as we are documenting them as public API

Reply:
Sounds good to me! Signing off 🙌

Reply:
@mnajdova We document these cases separately within the docs so that users are aware that they get access to these methods when they use `extendTheme` and not for simple object.